src/id.ts

```diff
@@ -32,6 +32,7 @@ function apply(apply: (op: Operation) => void, ed: Editor, op: Operation) {
     switch (op.type) {
     case 'insert_node':
         walk(op.node, el => {
+            el = { ...el }
```

Comment:
Change unrelated to this PR.

Reply:
The error I get without this change in some cases: 
TypeError: Cannot add property id, object is not extensible